frontend/src/components/cluster/Overview.tsx

```diff
@@ -116,15 +116,44 @@ function EventsSection() {
         columns={events ? [
           {
             label: 'Type',
-            getter: event => event.involvedObject.kind
+            getter: event => event.involvedObject.kind,
+            sort: (e1: Event, e2: Event) => {
+              if (e1.involvedObject.kind < e2.involvedObject.kind) {
+                return 1;
+              } else if (e1.involvedObject.kind > e2.involvedObject.kind) {
+                return -1;
+              } else {
+                return 0;
+              }
+            }
           },
           {
             label: 'Name',
             getter: event => event.involvedObject.name,
+            sort: (e1: Event, e2: Event) => {
+              const e1Name = e1.involvedObject.name.toUpperCase();
+              const e2Name = e2.involvedObject.name.toUpperCase();
```

Comment:
I don't think uppercasing is necessary. For the rest, please see my previous comments.